concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -309,9 +309,9 @@ jobs:
     plan:
       - get: pipeline-trigger
         trigger: true
-        passed: [pipeline-lock]
+        passed: ['pipeline-lock']
       - get: paas-cf
```

Comment:
Less is sometimes more. I'd normalise on format that doesn't add quotes around job names... (we don't have jobs that have space in name).